src/NuGet.Clients/NuGet.MSSigning.Extensions/NuGetMSSignCommand.resx

```diff
@@ -182,4 +182,24 @@ nuget mssign MyPackage.nupkg -Timestamper https://foo.bar -CertificateFile foo.p
   <data name="MSSignCommandOverwriteDescription" xml:space="preserve">
     <value>Switch to indicate if the current signature should be overwritten. By default the command will fail if the package already has a signature.</value>
   </data>
+  <data name="RepoSignCommandDescription" xml:space="preserve">
+    <value>Sign a NuGet package by adding repository signature with specified p7b file.</value>
+  </data>
+  <data name="RepoSignCommandOwnersDescription" xml:space="preserve">
+    <value>Semicolon separated list of package owners.</value>
+  </data>
+  <data name="RepoSignCommandUsageDescription" xml:space="preserve">
+    <value>Sign a NuGet package with repository signature.</value>
+  </data>
+  <data name="RepoSignCommandUsageExamples" xml:space="preserve">
+    <value>nuget reposign MyPackage.nupkg -Timestamper https://foo.bar -CertificateFile foo.p7b -CSPName "Cryptographic Service Provider"  -KeyContainer "4003d786-cc37-4004-bfdf-c4f3e8ef9b3a" -CertificateFingerprint "4003d786cc374004bfdfc4f3e8ef9b3a"  -Owners bar;foo -V3ServiceIndexUrl https://v3.index
```

Comment:
`nuget reposign MyPackage.nupkg -Timestamper http://timestamper.test
 -CertificateFile certificates.p7b -CSPName "Cryptographic Service Provider" -KeyContainer 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a -CertificateFingerprint 8599ADD1C62EE42E315EA887FF60908B7A7C6E9B -PackageOwners owner1;owner2 -V3ServiceIndexUrl https://v3.index.test`

Changed timestamper URL from https://foo.bar to http://timestamper.test.  We should avoid using "foobar."  Also, .test domains are good for tests and examples; they'll never resolve to a real domain.

Removed unnecessary spaces and quotes.

Changed the fingerprint to a SHA-1 fingerprint to avoid an implication that key container and fingerprint values are related.

Updated `PackageOwners` and `V3ServiceIndexUrl` arguments for above reasons.